crates/modelardb_compression/src/lib.rs

```diff
@@ -892,15 +1159,30 @@ mod tests {
 
     // Tests for can_models_be_merged().
     #[test]
-    fn test_models_can_be_merged() {
+    fn test_equal_models_without_residuals_can_be_merged() {
         assert!(can_models_be_merged(
             0,
             1,
             &UInt64Array::from_iter_values([1, 1]),
             &UInt8Array::from_iter_values([1, 1]),
             &ValueArray::from_iter_values([1.0, 1.0]),
             &ValueArray::from_iter_values([2.0, 2.0]),
-            &BinaryArray::from_iter_values([[1], [1]])
+            &BinaryArray::from_iter_values([[1], [1]]),
+            &BinaryArray::from_iter_values([[], []])
+        ))
+    }
+
+    #[test]
+    fn test_equal_models_second_residuals_can_be_merged() {
```

Comment:
Maybe rename to `test_equal_models_with_no_residuals_between_can_be_merged` or something similar.

Reply:
Could also be renamed to `test_equal_models_with_residuals_in_second_can_be_merged` to match `test_models_with_residuals_in_first_cannot_be_merged()`

Reply:
Fixed by replacing the name with the second suggestion.